sdk/core/core-client/src/operationHelpers.ts

```diff
@@ -103,3 +105,15 @@ function getPropertyFromParameterPath(
   }
   return result;
 }
+
+const operationRequestMap = new WeakMap<OperationRequest, OperationRequestInfo>();
+
+export function getOperationRequestInfo(request: OperationRequest): OperationRequestInfo {
+  let info = operationRequestMap.get(request);
+
+  if (!info) {
+    info = {};
+    operationRequestMap.set(request, info);
+  }
+  return info;
```

Comment:
Are you setting info anywhere else? it looks like it will always be `{}`?

Reply:
We assign properties to this object in `serviceClient.ts` lines 153 and 154.